paasta_tools/setup_marathon_job.py

```diff
@@ -512,7 +512,7 @@ def setup_service(service, instance, client, marathon_config,
 
     log.info("Setting up instance %s for service %s", instance, service)
     try:
-        complete_config = marathon_tools.create_complete_config(service, instance, marathon_config, soa_dir)
+        complete_config = marathon_tools.create_complete_config(service, instance, soa_dir)
```

Comment:
Wow, way for us to fail at dependency injection amirite?

Someone who knows more than me would probably say that we should go the other direction and *actually* pass in the config object instead of having it re-look it up. (and then junk the service/instance inputs because they are in the ojbect now thanks to you)